### DISCUSSION TITLE
Cannot insert the value NULL into column 'SatisId'

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5317] [Help]
![image](https://user-images.githubusercontent.com/24252615/102787937-e4fda580-43b2-11eb-8db7-8c2d18d07aa9.png)

ID not automatically added

**Comment by dfaruque** [comment]
is the column "SatisId" an identity column?

**Reply by havvanur160401046** [reply]
Yes . does not give automatic ids in all tables.

**Reply by edwardch** [reply]
Please can you provide more detail here so we can help? 
Like code in your xyzRow.cs, and maybe even in the migration if necessary.

**Comment by VictorTomaili** [comment]
It's might be identity but it's not an autoincrement column. 

[Sql Autoincrement](https://www.w3schools.com/sql/sql_autoincrement.asp)
